packages/esm-service-queues-app/src/queue-table/default-queue-table.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
```

Comment:
This file is just a copy of `active-visits-table.test.tsx`, with minor changes to test `<DefaultQueueTable>` instead and some minor verification tweaks.